src/Common.cpp

```diff
@@ -80,11 +86,12 @@ ResumeIdentificationToken ResumeIdentificationToken::fromString(
 }
 
 std::string ResumeIdentificationToken::toString() const {
```

Comment:
Added a test, rather not make this final for now in case @lehecka has plans. I just wanted it printable.